### DISCUSSION TITLE
[Projects Beta] Feature preview of colour-blind theme is not supported

### DISCUSSION THREAD

**Discussion by jaghaimo** [community/community#6363] [Projects and Issues] [answered]
This beta conflicts with the new colour-blind theme feature preview. List view does not respect theme at all, and table view has only some parts of the theme applied.

**Comment by WillsterJohnson** [comment]
Just discovered this after reporting [the same problem](https://github.com/github/feedback/discussions/6851), I can't believe it's been 2 weeks with no word on a CSS issue...

**Comment by mariorod** [comment, marked as answer]
Hi ... we are working on a fix and it should be out in production in the coming week.

**Reply by Mahinijas** [reply]
👍

**Reply by jpriestk999-tech** [reply]
can you help me out fixing my devices

**Comment by rubel726702-art** [comment]
rubel726702@gmail.com
